charts/k8s-service/values.yaml

```diff
@@ -372,6 +372,27 @@ additionalContainerEnv: {}
 #         envVarName: CONFIG_FOO
 configMaps: {}
 
+# persistentVolumes is a list of maps that specifies PeristantVolumes that should be mounted on the pod.
+# Each entry represents a persistent volume which should already exist within your cluster. The value is also a map and has the following attributes:
+#   - volumeName (string) (required)
+#     : The name of the Persistent Volume Claim on which this Persistent Volume in bound.
```

Comment:
update this description -- maybe something like "the logical name of the volume, as it will be registered in kubernetes"